symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/general-info.scss

```diff
@@ -1,3 +1,27 @@
-.organization-name-container{
+.organization-name-container {
   grid-column: span 2;
 }
+
+.no-of-employees-value {
+  font-family: "Nunito Sans";
```

Comment:
can't use genric scss vars here?

Reply:
avoid custom css as much as possible. with use of oxd components, you won't need these styles.